src/test/java/pages/LoginPage.java

```diff
@@ -0,0 +1,34 @@
+package pages;
+
+import org.apache.commons.lang3.RandomStringUtils;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class LoginPage {
+    AccountModel accountModel = new AccountModel();
+    private final WebDriver driver = WebDriverSingletion.getInstance().openBrowser();
+    private static final By USER_EMAIL_INPUT = By.cssSelector("#email_create");
+    private static final By CREATE_BUTTON = By.cssSelector("button[name=SubmitCreate]");
+    private static final By ACCOUNT_EMAIL = By.name("email");
+    private static final By ACCOUNT_PASSWORD = By.name("passwd");
+    private static final By SIGN_IN_BUTTON = By.cssSelector("button[name=SubmitLogin]");
+
+    private static final String URL = "http://automationpractice.com/index.php?controller=authentication&back=my-account";
+    private static final String USER_EMAIL = RandomStringUtils.randomAlphabetic(8) + "@gmail.com";
+
+    public LoginPage() {
+        driver.get(URL);
+    }
+
+    public void goToCreatingAnAccount() {
```

Comment:
Why void? I must remember that all public methods in page object must return current page or new page or any value, but not void